src/__tests__/snapshot-tests/__snapshots__/documenter.test.ts.snap

```diff
@@ -3575,6 +3575,28 @@ use the \`id\` attribute, consider setting it on a parent element instead.",
       "optional": true,
       "type": "string",
     },
+    {
+      "description": "Attributes to add to the native element.
+Some attributes will be automatically combined with internal attribute values:
+- \`className\` will be appended.
+- Event handlers will be chained, unless the default is prevented.
+
+We do not support using this attribute to apply custom styling.",
+      "inlineType": {
+        "name": "Omit<React.HTMLAttributes<HTMLElement>, "children"> & Record<\`data-\${string}\`, string>",
+        "type": "union",
+        "values": [
+          "Omit<React.HTMLAttributes<HTMLElement>, "children">",
+          "Record<\`data-\${string}\`, string>",
```

Comment:
I remember we got live build failures related to this syntax. Are they mitigated now?

UPD - I found it now, you use `@awsuiSystem` annotation on the imports to remove them. Still keeping this comment for posterity, it might be useful